.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -58,6 +58,9 @@ body:
     attributes:
       label: Ignition config
```

Comment:
Aside: should we change this to e.g. "Butane or Ignition config"?

Reply:
Good call.  Updated.